backend/flask/app.py

```diff
@@ -114,8 +174,9 @@ def create_link_token(userId):
             client_name="EconMetrics",
             country_codes=[CountryCode("US")],
             language='en',
+            # tokens are associated with indiviual users and therefore each user makes a unique request to plaid for link
             user=LinkTokenCreateRequestUser(
-                client_user_id=userId  # tokens are associated with indiviual users
```

Comment:
thanks for moving these comments to the lines before the code as opposed to in the same line!